src/omv/modules/py_sensor.c

```diff
@@ -451,6 +451,46 @@ static mp_obj_t py_sensor_get_rgb_gain_db() {
     });
 }
 
+static mp_obj_t py_sensor_set_auto_blc(uint n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
+    enum { ARG_enable, ARG_regs };
+    static const mp_arg_t allowed_args[] = {
+        { MP_QSTR_enable, MP_ARG_REQUIRED | MP_ARG_INT },
+        { MP_QSTR_regs,   MP_ARG_OBJ },
```

Comment:
Is `regs` required ? Seems so, it should be:
```
{ MP_QSTR_regs, MP_ARG_REQUIRED | MP_ARG_OBJ },
```
Or if it's not, then
```
{ MP_QSTR_regs, MP_ARG_OBJ, {.u_rom_obj = MP_ROM_NONE} },
```

Reply:
Fixed. Thanks for the catch.